Makefile

```diff
@@ -70,6 +74,8 @@ tools.timestamp:
 	go get -u github.com/vbatts/git-validation/...
 	go get -u github.com/goadesign/goa/goagen
 	go get -u github.com/haya14busa/goverage
+	go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
Humm wanna use it somewhere?

Reply:
```
api.go

  type idler struct {                                                                                                                                                                                              
    userIdlers      *openshift.UserIdlerMap                                                                                                                                                                        
    clusterView     cluster.View                                                                                                                                                                                   
    openShiftClient client.OpenShiftClient                                                                                                                                                                         
✗   controller      openshift.Controller         // not used anywhere                                                                                                                                                                   
    tenantService   tenant.Service                                                                                                                                                                                 
  }                                                                                                                                                                                                                
```

Reply:
yes right i mean in a makefile target like this for example : 

https://github.com/fabric8-services/fabric8-build-service/blob/master/Makefile#L176